python/cudf/cudf/tests/test_orc.py

```diff
@@ -1995,3 +2042,30 @@ def test_orc_reader_desynced_timestamp(datadir, inputfile):
     got = cudf.read_orc(path)
 
     assert_frame_equal(cudf.from_pandas(expect), got)
+
+
+@pytest.mark.parametrize("compression", ["LZ4", "SNAPPY", "ZLIB", "ZSTD"])
```

Comment:
could not include brotli, seems like pandas does not support it on the writer side.